src/ngx_http_lua_resolve.h

```diff
@@ -0,0 +1,18 @@
+
+/*
+ * Copyright (C) Sergey Kharkhardin (slimboyfat)
```

Comment:
You need to grant the copyright to the authors of this project before this can be merged. Thanks for your understanding.

Reply:
NP